force-app/main/default/classes/JobApplicationTriggerHandler.cls

```diff
@@ -77,4 +118,30 @@ public with sharing class JobApplicationTriggerHandler {
         });
         return statusToSubjectMap;
     }
+
+    private static List<Id> pluckAccountIds(List<JobApplication__c> JobApplications) {
```

Comment:
```suggestion
    private static List<Id> getAccountIdsFromJobApplication(List<JobApplication__c> JobApplications) {
```

Reply:
I love the encapulsation but I personally feel the method name is a little ambiguous on what it is suppose to be doing.

Reply:
That's a great point. If was able to update the method to pull AccountIds from any sObject list, 'pluckAccountIds' might be a valid, but for this scenario, I agree it should be more explicitly named